privileges-grantor/README.md

```diff
@@ -11,8 +11,7 @@ For more information please refer to the main [Apiary](https://github.com/Expedi
 | instance\_name | Privileges Grantor instance name to identify resources in multi-instance deployments. | string | `""` | no |
 | lambda\_timeout | The time after which the lambda execution stops. | string | `"200"` | no |
 | memory | The amount of memory (in MiB) to be used by Lambda | string | `"512"` | no |
-| metastore\_events\_filter | List of metastore event types to be added to SNS filter. Supported format: `[ "CREATE_TABLE","ALTER_TABLE" ]` | list | `[ "CREATE_TABLE", "ALTER_TABLE" ]` | no |
-| database\_filter | List of database names to be added to SNS filter. Supported format: `[ "DB_NAME_1", "DB_NAME_2" ]` | list | n/a | yes |
+| metastore\_events\_sns\_filter | SNS filter of message attributes to the added to the sns sns topic subscription. Supported format: SNS filter format | string | `{ "eventType": [ "CREATE_TABLE", "ALTER_TABLE" ] }` | no |
```

Comment:
```suggestion
| metastore\_events\_sns\_filter | SNS filter of message attributes to be added to the SNS topic subscription. Supported format: SNS filter format. | string | `{ "eventType": [ "CREATE_TABLE", "ALTER_TABLE" ] }` | no |
```

Reply:
"SNS filter format" -> can we link to some documentation describing what this filter format is?

Reply:
done